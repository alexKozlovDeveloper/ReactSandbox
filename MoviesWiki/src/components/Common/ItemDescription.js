import React, { Component } from "react";

function ItemDescription(props) {
    return  <div>
                {props.description}
            </div>;
}

export default ItemDescription;