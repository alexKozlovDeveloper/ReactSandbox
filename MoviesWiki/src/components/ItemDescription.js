import React, { Component } from "react";

//import styles from "../styles/ItemImage.css"

class ItemDescription extends Component {
    render() {
        
        //className={styles.imagecontainer}
        
        return (
            <div>
                {this.props.description}
            </div>
        );
    }
}

export default ItemDescription;