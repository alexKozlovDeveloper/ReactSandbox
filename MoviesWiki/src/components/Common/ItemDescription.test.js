import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import ItemDescription from "./ItemDescription";

configure({adapter: new Adapter()});

test('ItemDescription rendering test', () => {

    var description = "someDescription";

    const output = shallow(<ItemDescription description={description} />);
    expect(shallowToJson(output)).toMatchSnapshot();
 });

