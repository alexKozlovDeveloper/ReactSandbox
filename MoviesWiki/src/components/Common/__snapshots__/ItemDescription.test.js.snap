// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemDescription rendering test 1`] = `
<div>
  someDescription
</div>
`;
